plugins/blocks/metadataExport/metadataExportBlockPlugin.inc.php

```diff
@@ -0,0 +1,102 @@
+<?php
+
+/**
+ * @file plugins/blocks/metadataExport/MetadataExportBlockPlugin.inc.php
+ *
+ * Copyright (c) 2013-2015 Simon Fraser University Library
+ * Copyright (c) 2003-2015 John Willinsky
+ * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+ *
+ * @class MetadataExportBlockPlugin
+ * @ingroup plugins_blocks_metadataExport
+ *
+ * @brief Class for metadata export block plugin
+ *
+ */
+
+import('lib.pkp.classes.plugins.BlockPlugin');
+
+class MetadataExportBlockPlugin extends BlockPlugin {
+	
+	var $metadataExportPlugins;
+	
+	function MetadataExportBlockPlugin() {
+		$this->metadataExportPlugins = PluginRegistry::loadCategory('metadataExportFormats');
+		uasort($this->metadataExportPlugins, create_function('$a, $b', 'return strcmp($a->getDisplayName(), $b->getDisplayName());'));
+	}
+
+	/**
+	 * Determine whether the plugin is enabled. Overrides parent so that
+	 * the plugin will be displayed during install.
+	 */
+	function getEnabled() {
```

Comment:
I think the installed checks were borrowed from another plugin; if this plugin isn't to be active until the system is installed, remove them.